File: WledService.py
import json

import requests as requests


class WledService:

    def __init__(self, ip):
        self.ip = ip

    def init_triangle_for_detection(self, triangle):
        url = "http://" + self.ip + "/json/state"
        data = json.dumps(triangle, default=lambda o: o.__dict__, sort_keys=True, indent=4)
        response = requests.post(url=url, data=data)

        if response.status_code == 200:
            print("init started")

File: Service/WledService.cs
using System.Text;
using System.Text.Json;
using CWIllumigon.Domain;

namespace CWIllumigon.Service;

public class WledService
{
    private string Ip { get; }

    public WledService(string ip)
    {
        Ip = ip;
    }

    public void Show(Triangle triangle)
    {
        using var client = new HttpClient();

        var url = "http://" + Ip + "/json/state";
        var json = JsonSerializer.Serialize(triangle);
        var data = new StringContent(json, Encoding.UTF8, "application/json");

        var response = client.PostAsync(url, data).Result;

        Console.WriteLine(!response.IsSuccessStatusCode ? "Something went wrong" : "Yeah");
    }
}